Sources/Contentful/ImageOptions.swift

```diff
@@ -347,3 +345,37 @@ private struct ImageParameters {
     static let quality          = "q"
     static let progressiveJPG   = "fl"
 }
+
+
+// Use CGColor instead of UIColor to enable cross-platform compatibility: macOS, iOS, tvOS, watchOS.
+internal extension CGColor {
+
+    // If for some reason the following code fails to create a hex string, the color black will be
+    // returned.
+    internal func hexRepresentation() -> String {
+        let hexForBlack = "000000"
+        guard let colorComponents = components else { return hexForBlack }
+        guard let colorSpace = colorSpace else { return hexForBlack }
+
+        let r, g, b: Float
+
+        switch colorSpace.model {
+        case .monochrome:
+            r = Float(colorComponents[0])
+            g = Float(colorComponents[0])
+            b = Float(colorComponents[0])
```

Comment:
Also, just to be super picky about colors paces: Please use the luma way of calculating the grayscale: https://en.wikipedia.org/wiki/Grayscale#Luma_coding_in_video_systems in stead of a single color channel ...